Makefile

```diff
@@ -14,15 +14,16 @@ CONTROLLER_DIR=${ROOT_DIR}/../${AWS_SERVICE}-controller
 CONTROLLER_BOOTSTRAP=./bin/controller-bootstrap
 CODE_GEN_DIR=${ROOT_DIR}/../code-generator
 
-AWS_SDK_GO_VERSION=$(shell curl -H "Accept: application/vnd.github.v3+json" \
-    https://api.github.com/repos/aws/aws-sdk-go/releases/latest | jq -r '.tag_name')
-ACK_RUNTIME_VERSION=$(shell curl -H "Accept: application/vnd.github.v3+json" \
-    https://api.github.com/repos/aws-controllers-k8s/runtime/releases/latest | jq -r '.tag_name')
-TEST_INFRA_COMMIT_SHA=$(shell curl -H "Accept: application/vnd.github.v3+json" \
-    https://api.github.com/repos/aws-controllers-k8s/test-infra/commits | jq -r ".[0].sha")
+ACK_RUNTIME_VERSION:=$(or $(shell echo $(ACK_RUNTIME_VERSION)),$(shell curl -H "Accept: application/vnd.github.v3+json" \
+    https://api.github.com/repos/aws-controllers-k8s/runtime/releases/latest | jq -r '.tag_name'))
+AWS_SDK_GO_VERSION:=$(or $(shell echo $(AWS_SDK_GO_VERSION)),$(shell curl -H "Accept: application/vnd.github.v3+json" \
+    https://api.github.com/repos/aws/aws-sdk-go/releases/latest | jq -r '.tag_name'))
+TEST_INFRA_COMMIT_SHA:=$(or $(shell echo $(TEST_INFRA_COMMIT_SHA)),$(shell curl -H "Accept: application/vnd.github.v3+json" \
+    https://api.github.com/repos/aws-controllers-k8s/test-infra/commits | jq -r ".[0].sha"))
```

Comment:
Did you test that this works? What was your testing strategy?  I expect `echo $(ACK_RUNTIME_VERSION)` returning an empty string and setting that instead of shell command.

```suggestion
ACK_RUNTIME_VERSION:=$(or $(ACK_RUNTIME_VERSION),$(shell curl -H "Accept: application/vnd.github.v3+json" \
    https://api.github.com/repos/aws-controllers-k8s/runtime/releases/latest | jq -r '.tag_name'))
AWS_SDK_GO_VERSION:=$(or $(AWS_SDK_GO_VERSION),$(shell curl -H "Accept: application/vnd.github.v3+json" \
    https://api.github.com/repos/aws/aws-sdk-go/releases/latest | jq -r '.tag_name'))
TEST_INFRA_COMMIT_SHA:=$(or $(TEST_INFRA_COMMIT_SHA),$(shell curl -H "Accept: application/vnd.github.v3+json" \
    https://api.github.com/repos/aws-controllers-k8s/test-infra/commits | jq -r ".[0].sha"))
```

Reply:
Thank you, James!